tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -3336,9 +3486,9 @@ vlan/test_vlan_ping.py:
 #######################################
 voq:
   skip:
-    reason: "Cisco 8800 doesn't support voq tests"
+    reason: "Cisco 8800 doesn't support voq tests,except T2"
     conditions:
-      - "asic_type in ['cisco-8000']"
+      - "asic_type in ['cisco-8000'],not T2"
```

Comment:
Don't need to update this line, T2 is not in the list, which is expected already. Voq will run on T2